libswift/CMakeLists.txt

```diff
@@ -0,0 +1,142 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+function(add_libswift name module_files_var)
+  cmake_parse_arguments(LIBSWIFT
+                        ""
+                        "SWIFT_EXEC;BUILD_DIR"
+                        ""
+                        ${ARGN})
+
+  set(libswiftModules "SIL" "Optimizer")
+  find_program(swiftpm_executable "swift-build")
```

Comment:
Can you hide this behind CMAKE_BUILD_TYPE STREQUAL "Debug". That way when running in release, we do not do this.